docs/sources/experiments/algorithm.md

```diff
@@ -0,0 +1,182 @@
+
+#### Model Groups
+
+Each model is assigned a 'model group'. A model group represents a number of trained classifiers that we want to treat as equivalent by some criteria. By default, this is aimed at defining models which are equivalent across time splits, to make analyzing model stability easier. The experiment defines model groups by a static set of data about the model (classifier module, hyperparameters, feature list) and a user-supplied list of keys that must correspond to some key in the matrix metadata (See end of 'Retrieving Data and Saving Completed Matrix' section). This data is stored in the `results.model_groups` table, along with a `model_group_id` that is used as a foreign key in the `results.models` table.
+
+
+#### Model Hash
+Each trained model is assigned a hash, for the purpose of uniquely defining and caching the model. This hash is based on the training matrix metadata, classifier path, hyperparameters (except those which concern execution and do not affect results of the classifier, such as `n_jobs`), and the given project path for the Experiment. This hash can be found in each row of the `results.models` table. It is enforced as a unique key in the table.
+
+#### Global Feature Importance
+The training phase also writes global feature importances to the database, in the `results.feature_importances` table. A few methods are queried to attempt to compute feature importances:
+* The bulk of these are computed using the trained model's `._feature_importances_` attribute, if it exists.
+* For sklearn's `SVC` models with a linear kernel, the model's `.coef_.squeeze()` is used. 
+* For sklearn's LogisticRegression models, `np.exp(model.coef_).squeeze()` is used.
+* Otherwise, no feature importances are written.
+
+
+### Test Matrix
+For each test matrix, predictions, individual importances, and evaluation metrics are written to the database.
+
+#### Predictions
+The trained model's prediction probabilities (`predict_proba()`) are computed and saved for the test matrix. More specifically, `predict_proba` returns the probabilities for each label (false and true), but in this case only the probabilities for the true label are saved in the `results.predictions` table. The `entity_id` and `as_of_date` are retrieved from the matrix's index, and stored in the database table along with the probability score, label value (if it has one), as well as other metadata.
+
+### Individual Feature Importance
+Feature importances (of a configurable number of top features, defaulting to 5) for each prediction are computed and written to the `results.individual_importances` table. Right now, there are no sophisticated calculation methods are integrated into the experiment; simply the top 5 global feature importances for are copied to the`individual_importances`
```

Comment:
"are integrated" ➡️ "integrated"?

"for are" ➡️ "are"?

"the`individual_importances`" ➡️ "the `individual_importances` table."?